addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -241,6 +278,9 @@ MockServer.include({
                 return this._mockResPartnerGetMentionSuggestions(args);
             }
         }
+        if (args.method === 'get_channel_mention_suggestions') {
+            return this._mockResPartnerGetChannelMentionSuggestions(args);
```

Comment:
avoid forwarding this top level args, the mocked method shouldn't need to worry about how the mock server receives params, it should have similar params as the equivalent python method